scripts/run_app_paas.sh

```diff
@@ -17,6 +17,7 @@ function check_params {
 
 function configure_aws_logs {
   # create files so that aws logs agent doesn't complain
+  mkdir -p /home/vcap/logs/
```

Comment:
if the folder already exists won't this explode

Reply:
No

Reply:
okay